codechecker_lib/analysis_manager.py

```diff
@@ -155,7 +174,7 @@ def signal_handler(*arg, **kwarg):
     signal.signal(signal.SIGINT, signal_handler)
 
     # Remove characters which could cause directory creation problems.
-    no_spec_char_name = re.sub(r'[^\w\-_\. ]', '_', args.name)
+    no_spec_char_name = re.sub(r'[^\w\-_. ]', '_', args.name)
```

Comment:
This is not a functional change just a cleanup. Dot was *not* replaced befor this modification and it is replaced afterwards.